sycl/unittests/scheduler/SchedulerTestUtils.hpp

```diff
@@ -12,9 +12,10 @@
 #include <detail/queue_impl.hpp>
 #include <detail/scheduler/scheduler.hpp>
 
-#include <functional>
 // This header contains a few common classes/methods used in
 // execution graph testing.
+#include <functional>
+#include <gmock/gmock.h>
```

Comment:
I think the comment should be below the includes as it describes the `SchedulerTestUtils.hpp` file.

Reply:
You're right. Will fix.